e2e/tests/edit.js

```diff
@@ -23,6 +23,13 @@ describe('Edit Page', () => {
         it('should allow to switch tabs', async () => {
             await EditPage.gotoTab(2);
             assert.equal(await EditPage.getInputValue('average_note'), '3');
-        })
+        });
+
+        it('should keep DateInput value after canceled edit', async () => {
```

Comment:
@fzaninotto Here is the test. May look a bit strange, but the original `DateInput` can't pass this test.